alpakaConfig.cmake

```diff
@@ -675,6 +675,21 @@ IF(ALPAKA_ACC_GPU_CUDA_ENABLE)
                     LIST(APPEND CUDA_NVCC_FLAGS -Xcudafe --diag_suppress=esa_on_defaulted_function_ignored)
                 ENDIF()
 
+                # avoids warnings on host-device signature of 'std::__shared_count<>'
+                IF(CUDA_VERSION EQUAL 10.0)
+                    LIST(APPEND CUDA_NVCC_FLAGS "-Xcudafe --diag_suppress=2905")
+                ENDIF()
+
+                # avoids warnings on host-device signature of 'std::__shared_count<>'
+                IF(CUDA_VERSION EQUAL 10.1)
+                    LIST(APPEND CUDA_NVCC_FLAGS "-Xcudafe --diag_suppress=2912")
+                ENDIF()
+
+                # avoids warnings on host-device signature of 'std::__shared_count<>'
+                IF(CUDA_VERSION EQUAL 10.2)
+                    LIST(APPEND CUDA_NVCC_FLAGS "-Xcudafe --diag_suppress=2976")
```

Comment:
Yes it is doing what is should. `"-Xcudafe --diag_suppress=2976"` should be one parameter because this is then also avoiding that the order is changed. `--diag_suppress=2976` is the argument of `-Xcudafe`.

Reply:
somehow two cases crashed: https://travis-ci.org/ComputationalRadiationPhysics/alpaka/jobs/615075195#L5087 I will try it again without the quotes.